src/bridge/EthereumJSBridge.js

```diff
@@ -30,8 +30,8 @@ const toAccountOperation = (account: Account) => (tx: Tx): Operation => {
     hash: tx.hash,
     address: sending ? tx.to : tx.from,
     amount: (sending ? -1 : 1) * tx.value,
-    blockHeight: tx.block.height,
-    blockHash: tx.block.hash,
+    blockHeight: tx.block && tx.block.height,
+    blockHash: tx.block && tx.block.hash,
```

Comment:
note: we need to make all `blockHeight` and `blockHash` a `?` because it can be null